docs/dev/library.rst

```diff
@@ -99,6 +99,10 @@ synchronized (via load or store) with the database.
 
     .. automethod:: items
 
+    .. note::
+       The :py:meth:`Album.items` method is not inherited from
+       :py:meth:`LibModel.items` for historical reasons.
+
```

Comment:
Wouldn't this make more sense below with the `Album.items` docs?

Reply:
I think some kind of note here makes sense because it would be reasonable to assume methods documented in LibModel can also be used by its children, but that's not the case here.

Although the main explanation is left with the Album.items docs as you've expanded upon.

Reply:
Fair enough.